pkg/cloudprovider/providers/azure/azure_util.go

```diff
@@ -116,6 +125,204 @@ func (az *Cloud) getSecurityRuleID(securityRuleName string) string {
 		securityRuleName)
 }
 
+// returns the full identifier of a publicIPAddress.
+func (az *Cloud) getpublicIPAddressID(pipName string) string {
+	return fmt.Sprintf(
+		publicIPAddressIDTemplate,
+		az.SubscriptionID,
+		az.ResourceGroup,
+		pipName)
+}
+
+// select load balancer for the service in the cluster
```

Comment:
this seems specific to load balancers - why do we prefer to have it in azure_util instead of azure_loadbalancer?

Reply:
Valid point moved